scripts/bin/base_nas_fanout_init.py

```diff
@@ -64,4 +65,5 @@ def parse_config_file():
     while nas_if.nas_os_if_list() == None:
         sleep(1)
 
-    parse_config_file()
+    if not os.path.isfile(nocreate_file):
```

Comment:
Now in presence of /etc/opx/nas_if_nocreate this utility will be called but may wait indefinitely on the above "while" loop since no interfaces are created. Can you please confirm/clarify that.